GoogleForCreators/web-stories-wp
# Infer User Capabilities from REST API Responses

## Context



## Summary

Instead of using capabilities loaded from config provider, use hyperlinks in the rest api to workout permissions. 

## Relevant Technical Choices



## To-do



## User-facing changes

Remove menu items if you do have access to edit them.
<img width="258" alt="Screenshot 2021-08-05 at 13 04 19" src="https://user-images.githubusercontent.com/237508/128347082-db0c5f71-fbc3-4389-80cc-6831b6200dee.png">



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Create a story as an admin. Save Story.
2. Login as contributor / author, 
3. Go to dashboard. 
4. See limited options in menu and title of the story is not a hyperlink. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8276 
Fixes #8269
Fixes #8603
